src/interactions/keyInteraction.ts

```diff
@@ -1,7 +1,7 @@
 ///<reference path="../reference.ts" />
 
 module Plottable {
-export type KeyCallback = (keyCode: number) => void;
+export type KeyCallback = (keyCode: number, event?: KeyboardEvent) => void;
```

Comment:
This is a bit odd -- most of our `Interaction` callbacks don't expose the low-level event.